components/compass-runtime-agent/internal/kyma/secrets/repository.go

```diff
@@ -0,0 +1,100 @@
+// Package appsecrets contains components for accessing/modifying client secrets
```

Comment:
We could think of some common library as there is similar code in Application (the only difference is data type).

Reply:
Yes, in different PR